### PR TITLE
chore(demo): Amend parsing of mender client version for GUI env

### PR DESCRIPTION
**Pull request by lluiscampos**
Amends commit 2f0e44b63d94d53a0f18b02fafdf884fd29cbda9

### CODE REVIEW

File: demo [hunk 1]
```diff
@@ -165,7 +165,7 @@ env_setup() {
     # Parse the Mender-Artifact version used from the other-components.yml file's image tag
     export MENDER_ARTIFACT_VERSION=$(awk -F':' '/mendersoftware\/mender-artifact/ {print $3}' other-components.yml)
     # Parse the mender version from docker-compose.yml mender image's tag
-    export MENDER_VERSION=$(awk -F':mender-' '/mendersoftware\/mender-client/ {print $3}' docker-compose.client.yml | sed 's/mender-//')
+    export MENDER_VERSION=$(awk -F':' '/mendersoftware\/mender-client/ {print $3}' docker-compose.client.yml | sed 's/mender-//')
```
Comment (oleorhagen) [inline demo:168]:
```
awk -F':mender-' '/mendersoftware\/mender-client/ {print $2}' docker-compose.client.yml
```

Comment (oleorhagen) [inline demo:168]:
```suggestion
    export MENDER_VERSION=$(awk -F':'mender- '/mendersoftware\/mender-client/ {print $2}' docker-compose.client.yml)
```

**Review by oleorhagen — APPROVED** [review]
Pre-approving, in spite of previous shitty review...

Left a suggestion :)

**Review by mzedel — APPROVED** [review]
alternatively you could also pass in the entire version and I remove any extra prefixes until I find something that looks like a version...

**Comment by lluiscampos** [comment]
> alternatively you could also pass in the entire version and I remove any extra prefixes until I find something that looks like a version...

Thanks. But I'll keep it this way so that we can also easily fix the 3.3.x branch.

**Comment by lluiscampos** [comment]
@mender-test-bot cherry-pick to:
* 3.4.x
* 3.3.x

**Pull request merged**